packages/math-input/src/store/index.ts

```diff
@@ -21,31 +45,28 @@ export const createStore = () => {
                 swipeEnabled,
             },
             {
-                onSwipeChange: (dx) => {
-                    store.dispatch({
-                        type: "OnSwipeChange",
-                        dx,
-                    });
+                onSwipeChange: (dx: number) => {
+                    store.dispatch(onSwipeChange(dx));
```

Comment:
I wanted one place with all the actions, rather than having some in the actions file and some YOLO-ing it

Reply:
YDD.  My favourite type of engineering! (**Y**OLO **D**rive **D**evelopment)  😂